### ISSUE TITLE
v0.3.1 breaks meteor build

### ISSUE THREAD

**Issue by serkandurusoy** [guyonroche/exceljs#288]
As soon as I upgrade from `0.3.0` to `0.3.1`, my build throws with the following error (_reverting back fixes it_)

```sh
W20170316-08:18:14.034(0)? (STDERR) /xxxxxx/node_modules/promish/lib/promish-class.js:12
W20170316-08:18:14.066(0)? (STDERR)   class Promish extends Base {
W20170316-08:18:14.066(0)? (STDERR)   ^^^^^
W20170316-08:18:14.066(0)? (STDERR) 
W20170316-08:18:14.066(0)? (STDERR) SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
W20170316-08:18:14.067(0)? (STDERR)     at exports.runInThisContext (vm.js:53:16)
W20170316-08:18:14.067(0)? (STDERR)     at Module._compile (module.js:373:25)
W20170316-08:18:14.067(0)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
W20170316-08:18:14.067(0)? (STDERR)     at Module.load (module.js:343:32)
W20170316-08:18:14.068(0)? (STDERR)     at Function.Module._load (module.js:300:12)
W20170316-08:18:14.068(0)? (STDERR)     at Module.require (module.js:353:17)
W20170316-08:18:14.068(0)? (STDERR)     at require (internal/module.js:12:17)
W20170316-08:18:14.068(0)? (STDERR)     at Object.<anonymous> (/xxxxxx/node_modules/promish/lib/promish.js:4:18)
W20170316-08:18:14.068(0)? (STDERR)     at Module._compile (module.js:409:26)
W20170316-08:18:14.068(0)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
=> Exited with code: 1
```

**Comment by guyonroche** [comment]
@serkandurusoy big apologies for this - the node version I'm testing on doesn't give me this warning. I have just this minute published 0.4.0 which should fix your issue (i.e. I added 'use strict' to that file).
Let me know if this resolves your build issue

**Comment by serkandurusoy** [comment]
@guyonroche yes it does fix it with 0.4.0. Thank you!

**Comment by solshark** [comment]
Any chance to fix it for node 0.12.x?
